project/Dependencies.scala

```diff
@@ -899,6 +899,7 @@ object Dependencies {
       .exclude("org.apache.maven.wagon", "wagon-http-shared4")
       .exclude("org.apache.httpcomponents", "httpclient")
       .exclude("org.apache.httpcomponents", "httpcore"),
+    "org.apache.maven" % "maven-compat" % Versions.Maven,
```

Comment:
Bumping from `3.3.9` to `3.5.x` requires `maven-compat`. Note there is no `3.4.x` version of maven. See https://maven.apache.org/docs/3.5.0-alpha-1/release-notes.html#Why_not_Maven_3.4.0.3F .